packages/prisma-client/migrations-cli/README.md

```diff
@@ -0,0 +1,151 @@
+## Intro
+
+This is a drop-in replacement for Prisma's migration engine. It adds:
+
+- an ability to write a migration in TypeScript,
+- an ability to write a rollback for a migration (\* to be implemented).
```

Comment:
```suggestion
- an ability to write a rollback for data migration (to be implemented).
```

Reply:
We can support rollbacks for schema migrations too. Or do you think we shouldn't?

Reply:
just want to be explicit what's meant, in that case data and schema migration